webapp/src/pages/eventConfig/components/EventConfigComponent.js

```diff
@@ -111,329 +225,496 @@ class EventConfigComponent extends Component {
-              >
-                {t("Application Close")}
-              </label>
-
-              <div className="col-sm-4">
-                <DateTimePicker
-                  clearIcon={null}
-                  disableClock={true}
-                  onChange={e =>
-                    this.updateDateTimeEventDetails("application_close", e)
-                  }
-                  value={new Date(updatedEvent.application_close)} />
+  
+              {/* Email From */}
+              <div className={"form-group row"}>
+                <label className={"col-sm-2 col-form-label"} htmlFor="email_from">
+                  {t("Email From")}
+                </label>
+  
+                <div className="col-sm-10">
+                  <input
+                    onChange={e => this.updateEventDetails("email_from", e)}
+                    type="email"
+                    className="form-control"
+                    id="email_from"
+                    value={updatedEvent.email_from} />
+                </div>
               </div>
-
-            </div>
-            <div className={"form-group row"}>
-              <label
-                className={"col-sm-2 col-form-label"}
-                htmlFor="review_open">
-                {t("Review Open")}
-              </label>
-
-              <div className="col-sm-4">
-                <DateTimePicker
-                  clearIcon={null}
-                  disableClock={true}
-                  onChange={e =>
-                    this.updateDateTimeEventDetails("review_open", e)}
-                  value={new Date(updatedEvent.review_open)} />
+  
+              {/* URL */}
+              <div className={"form-group row"}>
+                <label className={"col-sm-2 col-form-label"}
+                  htmlFor="url">
+                  {t("URL")}
+                </label>
+                <div className="col-sm-10">
+                  <input
+                    onChange={e => this.updateEventDetails("url", e)}
+                    type="text"
+                    className="form-control"
+                    id="url"
+                    value={updatedEvent.url} />
+                </div>
               </div>
 
-              <label
-                className={"col-sm-2 col-form-label"}
-                htmlFor="review_close">
-                {t("Review Close")}
-              </label>
-
-              <div className="col-sm-4">
-                <DateTimePicker
-                  clearIcon={null}
-                  disableClock={true}
-                  onChange={e =>
-                    this.updateDateTimeEventDetails("review_close", e)}
-                  value={new Date(updatedEvent.review_close)} />
+                 
+              <hr style={{ "marginTop": "50px" }}></hr>
+  
+              {/* UTC Time request */}
+              <div className="col-md-12 time-warning">
+                      <span class="badge badge-warning">{t('Times will automatically convert to UTC')}</span>
```

Comment:
I don't think it actually *converts* to UTC, it just treats whatever time you enter as UTC? Hard for me to test because I'm currently in GMT = UTC :)

Reply:
Hi, Im a little puzzled here. 

So the UI will for instance be set to 10:00 am and thats what the UI will show, in the state the time will reflect 8:00 am in  my particular instance because South African Standard Time is 2hours ahead of UTC.

So it appears as if behind the scenes in the state the user input time will be adapted to UTC as in my case, does this make sense?

Reply:
I can join the two components for now if you like, and you can make it work later?

Reply:
Perhaps we need to adapt the pill that says `Times will automatically convert to UTC`

Reply:
Ah got it, that's cool. So it make it more clear lets change the text to something like "Enter times in your local time zone, these will automatically be converted to UTC"